src/components/Articles/SingleArticle.js

```diff
@@ -24,8 +26,11 @@ const SingleArticle = ({ article }) => (
           </div>
           <div className="row">
             <div className="col-md-3 article-time">
-              {articleCreated(article.createdAt)}
-              &nbsp;&nbsp;|&nbsp;3 Min
+              <p>
+                <span>{articleCreated(article.createdAt)}</span>
+                |
+                <span>3 Min Read</span>
```

Comment:
Is it possible to put the actual reading time here?

Reply:
I had added the `reading_time` thanks for noticing that.